tools/downloader/README.md

```diff
@@ -379,6 +381,69 @@ Toolkit will still be created, so that you can inspect it.
 See the "Shared options" section for information on other options accepted by
 the script.
 
+## Networks compiler usage
+The basic usage is to run the script like this:
+
+```sh
+./compiler.py --all
+```
+
+This will compile all models for which it is supported. Other models
+are ignored.
+
+The `--all` option can be replaced with other filter options to compile only
+a subset of models. See the "Shared options" section.
+
+The current directory must be the root of a tree of model files created by the model
+converter. To specify a different model tree path, use the `--model_dir` option:
+
+```sh
+./compiler.py --all --model_dir my/model/directory
+```
+
+By default, the compiled models are placed into the same model tree. To place them
+into a different directory tree, use the `-o`/`--output_dir` option:
+
+```sh
+./compiler.py --all --output_dir my/output/directory
+```
+
+By default, the script will compile models in FP16 and FP32 precisions.
+To only produce models in a specific precision, use
+the `--precisions` option:
+
+```sh
+./compiler.py --all --precisions=FP32
+```
+
+The script will attempt to locate Compile Tool using
+the environment variables set by the OpenVINO&trade; toolkit's `setupvars.sh`/`setupvars.bat`
+script. You can override this heuristic with the `--compiler` option:
+
+```sh
+./compiler.py --all --compiler my/openvino/path/compile_tool/compile_tool_executable
+```
+
+It's possible to specify a target device for Compile Tool
+to compile for, by using the `--target_device` option:
+
+```sh
+./compiler.py --all --target_device MYRIAD
+```
+
+The supported values are those accepted by the "target_device" option in
+Compile Tool documentation. If this option is unspecified, default is `MYRIAD`.
+
```

Comment:
You're missing the documentation for the other options, `--input_precision` and `--output_precision`.

Reply:
Those arguments were replaced with another one, new one is added to the documentation.

Reply:
They still exist in the code, though.